app/integration_test/app_test.dart

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2022 Sharezone UG (haftungsbeschränkt)
+// Licensed under the EUPL-1.2-or-later.
+//
+// You may obtain a copy of the Licence at:
+// https://joinup.ec.europa.eu/software/page/eupl
+//
+// SPDX-License-Identifier: EUPL-1.2
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:sharezone/main/run_app.dart';
+import 'package:sharezone/main/sharezone.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  AppDependencies dependencies;
+  _UserCredentials user1;
+
+  setUpAll(() async {
+    dependencies = await initializeFlutterApp(isIntegrationTest: true);
```

Comment:
No problem for now but does it fundamentally even make sense that the app **returns** dependencies? Wouldn't we pass dependencies to the app to run?   

I mean it might make sense to pass dependencies and still have a way to access the objects created with those dependencies, I guess? Idk. 

Like I said it's okay for now, but it kinda confuses me.

Reply:
Yeah, we can definitely rework this, but it requires some refactoring in the `main.dart` because we do so many dirty things in the `runFlutterApp` method 😅 Therefore, my plan is tough in the app the thing which are blocking the integration tests, write a lot of integration tests and then start refactoring.